### PR TITLE
Update translation in french

### PR DESCRIPTION
**Pull request by forresst**
1. Fix 'Links' to 'Liens'
2. Improve server middleware code example (parcel-bundler/website@6156d395114837cd581a541a4d2312a17146bd30)
3. Fix some commands in cli (parcel-bundler/website@03557d5d44e85fd9f3c3e8ae68abad655ed7132f)
4. Add https flag to cli (parcel-bundler/website@3acca7a1147f175944c8c316cf50556d8fab2bba)
5. Fix typo in cli.md (parcel-bundler/website@8edd475d0b7300a10ac3509b62889a398cf327fd)
6. cssnano config doc (parcel-bundler/website@9b346cc5deaea0d8af60b4794df3f9b99d297e1d)
7. Note that HMR can fail based on editor save-mode (parcel-bundler/website@cac896ddad92b53122fb4ae5712f248c5e10acef)

### CODE REVIEW

File: src/i18n/fr/docs/cli.md [hunk 1]
```diff
@@ -144,14 +144,26 @@ Disponible dans : `build`
 parcel build entry.js --detailed-report
 ```
 
-### Définir un certificat personnalisé
+### Désactive https
```
Comment (fathyb) [inline src/i18n/fr/docs/cli.md]: Désactive**r**

File: src/i18n/fr/docs/cli.md [hunk 2]
```diff
@@ -144,14 +144,26 @@ Disponible dans : `build`
...
 Disponible dans : `serve`
 
 ```bash
-parcel build entry.js --cert certificate.cert --key private.key
+parcel build entry.js --https
+```
+
+⚠️ Cet indicateur génère un certificat auto-signé. Vous devrez peut-être configurer votre navigateur pour autoriser les certificats auto-signés pour localhost.
+
+### Définit un certificat personnalisé
```
Comment (fathyb) [inline src/i18n/fr/docs/cli.md]: Définir

File: src/i18n/fr/docs/api.md [hunk 3]
```diff
@@ -124,12 +124,19 @@ Un exemple d'utilisation du middleware de Parcel avec express
...
+
+  // Initialise un nouveau bundler en utilisant un fichier et des options
+  const bundler = new Bundler(file, options);
+
+  // express utilise le middelware de bundler, cela permettra à Parcel de gérer chaque requête sur votre serveur express
+  app.use(bundler.middleware());
 
-// express utilise le middleware de bundler, cela permettra à Parcel de gérer chaque requête sur votre serveur express
-app.use(bundler.middleware());
+  // Ecoute du port 8080
```
Comment (fathyb) [inline src/i18n/fr/docs/api.md]: Écoute

File: src/i18n/fr/docs/api.md [hunk 4]
```diff
@@ -124,12 +124,19 @@ Un exemple d'utilisation du middleware de Parcel avec express
...
-// Initialise un nouveau empaqueteur (bundler) en utilisant un fichier et des options (pour les options et le fichier, voir la documentation du bundler)
-const bundler = new Bundler(file, options);
+async function start() {
+  const file = 'index.html'; // Passe un chemin absolu vers le point d'entrée ici
+  const options = {}; // Voir la section des options de la doc de l'api, pour les possibilités
+
+  // Initialise un nouveau bundler en utilisant un fichier et des options
+  const bundler = new Bundler(file, options);
+
+  // express utilise le middelware de bundler, cela permettra à Parcel de gérer chaque requête sur votre serveur express
```
Comment (fathyb) [inline src/i18n/fr/docs/api.md]: > middelware **du** bundler

Hmm, j'ai l'impression qu'on modifie le sens de la phrase en ignorant `Let` dans `Let express use the bundler middleware, ...`.

Tu pense quoi de `Laisse express utiliser le middelware du bundler, ...`, ou `Permet express d'utiliser...`?

**Comment by forresst** [comment]
@fathyb Thanks, i fixed your remarks

**Pull request merged**